src/Model/Contact.php

```diff
@@ -1851,9 +1851,8 @@ private static function updateFromProbeArray(int $id, array $ret)
 			$ret['prv'] = false;
 			$ret['contact-type'] = $ret['account-type'];
 			if ($ret['contact-type'] == User::ACCOUNT_TYPE_COMMUNITY) {
-				$apcontact = APContact::getByURL($ret['url'], false);
-				if (isset($apcontact['manually-approve'])) {
-					$ret['forum'] = (bool)!$apcontact['manually-approve'];
+				if (isset($ret['manually-approve'])) {
```

Comment:
Please simplify the conditional to a single level:
```php
if ($ret['contact-type'] == User::ACCOUNT_TYPE_COMMUNITY && isset($ret['manually-approve'])) {
```

Reply:
Done